packages/node/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -50,8 +50,17 @@ import { KeyObject } from "crypto";
 import { fetch as globalFetch } from "cross-fetch";
 
 import { EventEmitter } from "events";
+import { Headers as NodeHeaders } from "node-fetch";
```

Comment:
We don't need the extra "node-fetch" dependency.

```suggestion
import { fetch as globalFetch } from "cross-fetch";

import { EventEmitter } from "events";
import { Headers as NodeHeaders } from "node-fetch";
```
```suggestion
import { fetch as globalFetch, Headers as NodeHeaders } from "cross-fetch";

import { EventEmitter } from "events";
```